### DISCUSSION TITLE
Which Raspberry Pi Pico pins do I have to connect SDA and SCL to?

### DISCUSSION THREAD

**Discussion by jonashuberts** [moononournation/Arduino_GFX#55] [Q&A] [answered]
I have a round1.28 TFT LCD GC9A01 Display. I would like to use it with my Raspberry Pi Pico as a Clock. But I don't know to which pins I Must connect SDA and SCL.

I also don't know which Board driver I have to use for the Pico, the driver from Earle F. Philhower or the official from Arduino.

I would be very happy if you could help me further

![Unbenannt2](https://user-images.githubusercontent.com/66276492/118939589-22f5fa00-b950-11eb-866b-b47919af9766.JPG)
![IMG_5156](https://user-images.githubusercontent.com/66276492/118942572-263eb500-b953-11eb-879f-1e94f192aee6.JPG)
![IMG_5155](https://user-images.githubusercontent.com/66276492/118942581-28a10f00-b953-11eb-878e-44901791e885.JPG)
![IMG_5154](https://user-images.githubusercontent.com/66276492/118942592-2b036900-b953-11eb-9184-92759fecc21f.JPG)

**Comment by moononournation** [comment, marked as answer]
* Raspberry Pi Pico dev board : TFT_CS: 17, TFT_DC: 27, TFT_RST(RES): 26, TFT_BL(BLK): 28
TFT_SCK(SCL): 18, TFT_MOSI(SDA): 19

All of the above is GPIO number(GPxx). I know the board did not have GPIO number is a difficulty, you may find it on the web:
https://www.google.com.hk/search?q=raspberry+pi+pico+pinout&ie=UTF-8&oe=UTF-8&hl=zh-hk&client=safari

**Reply by jonashuberts** [reply]
Thanks for your fast Answer.  
I have connected the pins as you suggested, but the Display is still leaving black

**Reply by moononournation** [reply]
https://youtu.be/bg1ANWpMtsw
